tests/models/test_ngsi_v2_subscriptions.py

```diff
@@ -147,6 +147,29 @@ def compare_dicts(dict1: dict, dict2: dict):
         with self.assertRaises(ValidationError):
             sub.throttling = 0.1
 
+    def test_model_dump_json(self):
```

Comment:
@tstorek most of the `tests.models` now work again. However, there is still an error probably caused by the `model_dump_json()`. It seems like the exclusion parameters are not working. Can you try this test?